src/AzureExtension/Services/DevBox/DevBoxManagementService.cs

```diff
@@ -138,12 +173,24 @@ public async Task<List<DevBoxProjectAndPoolContainer>> GetAllProjectsToPoolsMapp
 
         await Parallel.ForEachAsync(projects.Data!, async (project, token) =>
         {
+            if (await FilterOutProjectAsync(project, developerId))

```

Comment:
>await [](http://example.com/codeflow?start=16&length=5)

curious why this has to async?

i assume it does, but that seems silly to me.

Reply:
It doesn't; just copied the old pattern. Changed.